<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="LAPTOP-K0FI96QS" failures="1" tests="5" name="Test" time="83.911" errors="0" timestamp="2025-09-02T12:53:51 IST">
  <testcase classname="TestCases.TC_001_AccountRegistration" name="verifyaccountregistration" time="12.175"/>
  <testcase classname="TestCases.TC_002_AccountLogin" name="verifylogin" time="11.559">
    <failure type="java.lang.AssertionError" message="Valid credentials should login successfully expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Valid credentials should login successfully expected [true] but found [false]
at TestCases.TC_002_AccountLogin.verifylogin(TC_002_AccountLogin.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verifylogin -->
  <testcase classname="TestCases.TC_002_AccountLogin" name="verifylogin" time="7.74"/>
  <testcase classname="TestCases.TC_002_AccountLogin" name="verifylogin" time="8.241"/>
  <testcase classname="TestCases.TC_002_AccountLogin" name="verifylogin" time="7.318"/>
  <testcase classname="TestBase.BaseClass" name="@BeforeSuite reportSetup" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite reportSetup -->
  <testcase classname="TestBase.BaseClass" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
</testsuite> <!-- Test -->
